README.md

```diff
@@ -57,14 +57,14 @@ It provides a suite of functionality that empowers you to create complex, robust
 
 ### Transactional Steps
 
-The first and foundational feature `acidic_job` provides is the `idempotently` method, which takes a block of transactional step methods (defined via the `step`) method:
+The first and foundational feature `acidic_job` provides is the `with_acidity` method, which takes a block of transactional step methods (defined via the `step`) method:
 
 ```ruby
 class RideCreateJob < ActiveJob::Base
   include AcidicJob
 
   def perform(ride_params)
-    idempotently with: { user: current_user, params: ride_params, ride: nil } do
+    with_acidity with: { user: current_user, params: ride_params, ride: nil } do
```

Comment:
I forgot to put this in the issue, but I think it also makes sense to change this named param from `with` to `given`.